tokens.xml

```diff
@@ -7073,7 +7073,7 @@ Equip {1}</text>
                 <cmc>0</cmc>
                 <pt>1/1</pt>
             </prop>
-            <set picURL="https://img.scryfall.com/cards/large/front/6/a/6aaa8539-8d21-4da1-8410-d4354078390f.jpg">AKH</set>
+            <set picURL="https://c1.scryfall.com/file/scryfall-cards/large/front/6/a/6aaa8539-8d21-4da1-8410-d4354078390f.jpg?1562844799">AKH</set>
```

Comment:
this has been coming for a while but every single image that starts with img.scryfall is broken

Reply:
see https://github.com/Cockatrice/Magic-Token/issues/92

Reply:
It looks like it could be fixed with automation. 

Change img to c1 and add in file/scryfall- after com/

As far as I can tell the ?number at the end of the link affects nothing, and the link works the same without it from what I can see.

Reply:
we have had one situation where the ? actually did affect the shown picture in scryfall over on the spoilers repo, the spoilers repo used to trim it but we reverted that

Reply:
The official info from Scryfall support is that they use the integer to bust the CND caches for various reasons.
One should use the URI they provide with the integer parameter at the end to ensure one gets the best version of the image currently available.

Not so sure that is super relevant for us as we cache pictures locally anyway and the client never realizes that a parameter got updated to redownload the file and update the local version.